app/pages/project/components/ProjectNavbar/ProjectNavbar.jsx

```diff
@@ -1,76 +1,57 @@
 import _ from 'lodash';
-import getRenderedSize from 'react-rendered-size';
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import withSizes from 'react-sizes';
+import sizeMe from 'react-sizeme'
 import ProjectNavbarNarrow from './components/ProjectNavbarNarrow';
-import ProjectNavbarWide from './components/ProjectNavbarWide';
+import ProjectNavbarWide, { SizeAwareProjectNavbarWide } from './components/ProjectNavbarWide';
 
 function haveNavLinksChanged(oldProps, newProps) {
-  const oldNavLinks = _.map(oldProps.navLinks, 'label');
-  const newNavLinks = _.map(newProps.navLinks, 'label');
-  return _.difference(newNavLinks, oldNavLinks).length > 0 || oldNavLinks.length !== newNavLinks.length;
-}
-
-function hasTitleChanged(oldProps, newProps) {
-  return oldProps.projectTitle !== newProps.projectTitle;
+  const oldLinks = _.map(oldProps.navLinks, 'label');
```

Comment:
I think this would be much easier to read if it used `Array.map`.
`const oldLinks = oldProps.navLinks.map(link => link.label);`